src/lib/clock.js

```diff
@@ -0,0 +1,14 @@
+import moment from 'moment-timezone';
+
+//initialization code
+setTimeout(() => {
+    //TODO: connect email system here
+    //email Students
+    //setInterval(/*Email Students*/, 86400000);
+}, (moment.tz(Date.now(), "America/Chicago").hour(16).minute(30).second(0).format('X')
+ - moment.tz(Date.now(), "America/Chicago").format('X')) * 1000);
+
+export function isBeforeNow(timestamp){
+    let currentTime = moment.tz(Date.now(), "America/Chicago");
+    return timestamp < parseInt(currentTime.format('X'));
```

Comment:
Why not just use ~`return timestamp < Date.now() / 1000`?

Moment seems like overkill for this function, and timezones are completely unnecessary when using unix timestamps.